### PR TITLE
Fix replacement of variable in Keycloak template

### PR DESCRIPTION
**Pull request by jirikuncar**

### CODE REVIEW

**Comment by rokroskar** [comment]
the problem seems to be that the travis user has uid 2000 but the jboss user inside the keycloak container has uid 1000 so it doesn't have permission to write to the mounted directory

File: docker-compose.yml [hunk 1]
```diff
@@ -17,6 +17,8 @@
 # limitations under the License.
 
 version: '3'
+volumes:
+  keycloak-data: {}
```
Comment (rokroskar) [inline docker-compose.yml]: this just makes the volume to be mounted as root and the file write fails again:

```
[jboss@57bab8d1bca9 ~]$ ls -la /data
total 72
drwxr-xr-x 2 root  root   4096 Sep 18 19:17 .
drwxr-xr-x 1 root  root   4096 Sep 18 19:17 ..
-rwxr-xr-x 1 jboss jboss  1168 Sep 18 14:53 docker-entrypoint.sh
-rw-r--r-- 1 jboss jboss 59979 Sep 18 08:46 renga-realm.json.tpl
```

**Comment by rokroskar** [comment]
I know 503ad6a is not the dynamic configuration we all want for keycloak and the user will be required to start the platform with `make start` (unless they create their own keycloak configuration) -- however, until we have clarified why docker mounts directories under different uids depending on the host system, I think this is the safe option to avoid failing builds elsewhere.

File: docker-compose.yml [hunk 2]
```diff
@@ -56,9 +55,7 @@ services:
       - "-Dkeycloak.migration.file=/data/renga-realm.json"
       - "-Dkeycloak.migration.strategy=IGNORE_EXISTING"
     volumes:
-      - ./services/keycloak/docker-entrypoint.sh:/data/docker-entrypoint.sh:ro
-      - ./services/keycloak/renga-realm.json.tpl:/data/renga-realm.json.tpl:ro
-      - ./services/keycloak/data:/data:rw
+      - ./services/keycloak/renga-realm.json:/data/renga-realm.json
```
Comment (leafty) [inline docker-compose.yml]: Not read-only?

Comment (rokroskar) [reply docker-compose.yml]: actually I couldn't see any difference on the resulting file permissions when I used the flags in docker-compose

Comment (leafty) [reply docker-compose.yml]: This does not necessarily translates into file permissions, but the volume driver will effectively forbid writting of the file

**Pull request merged**